src/main/java/step5/domain/LottoNumber.java

```diff
@@ -0,0 +1,46 @@
+package step5.domain;
+
+import java.util.Objects;
+
+public class LottoNumber {
+
+    private static final Integer MIN = 1;
+    private static final Integer MAX = 45;
+    private final Integer lottoNumber;
+
+    public LottoNumber(String lottoNumber) {
+        this(Integer.parseInt(lottoNumber));
+    }
+
+    public LottoNumber(Integer lottoNumber) {
+        checkValidRange(lottoNumber);
+        this.lottoNumber = lottoNumber;
+    }
+
+    private void checkValidRange(Integer lottoNumber) {
+        if (lottoNumber < MIN || lottoNumber > MAX) {
+            throw new IllegalArgumentException("로또 번호는 1 ~ 45 사이 값이어야 합니다.");
```

Comment:
```suggestion
            throw new IllegalArgumentException("로또 번호는 " + MIN + " ~ " + MAX + " 사이 값이어야 합니다.");
```

예외메시지에서도 상수를 사용한다면 조건이 변경 됐을 때 메시지를 신경쓰지 않아도 되겠네요!